app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,370 @@
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="341dp"
+            android:text="@string/Casts"
+            android:textAlignment="textStart"
+            android:textColor="@color/ECECEC"
+            android:textSize="14sp"
+            android:textStyle="normal|bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/storyline_text_textView" />
+
+        <ImageView
+            android:id="@+id/actor_imageView"
+            android:layout_width="80dp"
+            android:layout_height="80dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="7dp"
+            android:contentDescription="@string/actor_name"
+            android:visibility="visible"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/casts_textView"
+            app:srcCompat="@drawable/downey"
+            tools:srcCompat="@drawable/downey" />
+
+        <ImageView
+            android:id="@+id/actor_imageView2"
+            android:layout_width="80dp"
+            android:layout_height="80dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="7dp"
+            android:contentDescription="@string/actor_name2"
+            android:visibility="visible"
+            app:layout_constraintStart_toEndOf="@+id/actor_imageView"
+            app:layout_constraintTop_toBottomOf="@+id/casts_textView"
+            app:srcCompat="@drawable/evans"
+            tools:srcCompat="@drawable/evans" />
+
+        <ImageView
+            android:id="@+id/actor_imageView3"
+            android:layout_width="80dp"
+            android:layout_height="80dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="7dp"
+            android:contentDescription="@string/actor_name3"
+            android:visibility="visible"
+            app:layout_constraintStart_toEndOf="@+id/actor_imageView2"
+            app:layout_constraintTop_toBottomOf="@+id/casts_textView"
+            app:srcCompat="@drawable/ruffalo"
+            tools:srcCompat="@drawable/ruffalo" />
+
+        <ImageView
+            android:id="@+id/actor_imageView4"
+            android:layout_width="80dp"
+            android:layout_height="80dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="7dp"
+            android:contentDescription="@string/actor_name4"
+            android:visibility="visible"
+            app:layout_constraintStart_toEndOf="@+id/actor_imageView3"
+            app:layout_constraintTop_toBottomOf="@+id/casts_textView"
+            app:srcCompat="@drawable/hemsworth"
+            tools:srcCompat="@drawable/hemsworth" />
+
+        <TextView
+            android:id="@+id/actorName_textView"
+            android:layout_width="80dp"
+            android:layout_height="32dp"
```

Comment:
android:layout_height="32dp" -> to wrap_content
here and below

Reply:
Done